src/ShearStressTransportEquationSystem.C

```diff
@@ -607,6 +622,119 @@ ShearStressTransportEquationSystem::compute_f_one_blending()
   fOneBlend.modify_on_device();
 }
 
+/** Compute mixing length for 2003 SST implementation
+ */
+void
+ShearStressTransportEquationSystem::compute_lt()
```

Comment:
One thing I am worried about... Do you think we can make `lt` optional? I would assume that for a lot of SST runs we don't need to extra things. What do you think? I guess we've replaced `gamma` which used to be an in place calculation with a field.

Reply:
Sounds good. I switched `gamma` and `lt` back to in place.